Makefile.common

```diff
@@ -442,7 +442,7 @@ endif
 
 # Shaders
 DIST_FILES_SHADERS=
-ifdef USE_OPENGL_SHADERS
+ifneq ($(USE_OPENGL_SHADERS)$(USE_GLES2),)
```

Comment:
Interesting. I didn't know this but apparently it is _the way_ according to [the GNU make docs](https://www.gnu.org/software/make/manual/html_node/Conditional-Syntax.html)

The docs also show usage of `strip` to remove white space, as otherwise whitespace values are interpreted as non-empty. Not sure if it is needed here.

I'm not great with make and so the intention wasn't obvious to me at all. Perhaps a comment might be helpful?

Reply:
Thanks for the proposition.
I've just added a comment.